ganga/GangaDirac/BOOT.py

```diff
@@ -42,16 +49,81 @@ def diracAPI(cmd, timeout=60, cred_req=None):
 
 #\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/#
 
+running_dirac_process = False
+dirac_process = None
+dirac_process_ids = None
+def startDiracProcess():
+    '''
+    Start a subprocess that runs the DIRAC commands
+    '''
+    HOST = '127.0.0.1'  #Connect to localhost
```

Comment:
Would it be better to use the name "localhost" instead? In that case you are not forcing IPv4.